### PR TITLE
Enumerable feature branch 👨‍💻 👨‍💻 

### PR DESCRIPTION
**Pull request by tsheporamantso**
## Enumerable feature branch 👨‍💻 👨‍💻 
On this milestone, we've managed to implement the following 🧠 🧠 
- [ ] Use the ruby-style guide in the project ✔️ 
- [ ] Created `MyList ` and `MyEnumerable` separate files 📁 
- [ ] Add ``methods to `class` and `module` ✔️ 
- [ ] Add README file with project-related information 📁 
- [ ] Add Linters and ensure there are linter errors 🤖 
- [ ] Followed correct `Gitflow` and `Ruby` best practices 🚥

### CODE REVIEW

File: my_list.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+class Mylist
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```
Comment (ShadyShawkat) [inline my_list.rb]: - [ ] Kindly make sure to implement this method to yield list members from the list. You've done so but using `each`. While this is correct, the purpose of this method is to learn about `yield` and how it works in a method. You can do something like the following to replicate the `each` behavior without actually using it.

```ruby
def each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end
```

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @tsheporamantso, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: my_list.rb (hunk 1 above)
Comment (tsheporamantso) [reply my_list.rb]: @ShadyShawkat  Thank you so much for such an insightful review, We will implement the changes as recommended.

**Review by skyv26 — APPROVED** [review]
## Hi @tsheporamantso and @erickma1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implemented the features as per previous code reviewers recommendation ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**